src/marine/smos_sss2ioda.py

```diff
@@ -14,7 +14,7 @@
 import netCDF4 as nc
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"../lib/pyiodaconv"
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@
```

Comment:
Missing a double quote at the end of the line.

Reply:
Sorry, I sent this at the same time you pushed the fix. Thanks for taking care of this!

Reply:
@srherbener no issues.
Please check now.